Bottom: d7ecec9a15aad095efa91b16c322b1c2955b944a
Top:    967bca733901552dbc152c959a410c5ebe950563
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-06 17:22:16 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index ed10b50..79a914a 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
